### PR TITLE
Firewall should allow 443 on management interface when pre-registration ...

### PR DESCRIPTION
**Pull request by dwlfrth**
## Description

Punch a hole in the iptables firewall of PacketFence to allow the httpd.portal server to be reach on the management ip, port 443, when pre-registration is enabled.
## Impacts

Shouldn't have any...

### CODE REVIEW

**Comment by julsemaan** [comment]
I honnestly think it should always be enabled.
Since that interface can now be used for the web auth portal, I don't see why we should still refuse access to it...

**Comment by dwlfrth** [comment]
You are kind of right but for the moment it is not.
Maybe we can simply merge that one and work on getting iptables rules fixed with another branch

File: NEWS.asciidoc [hunk 1]
```diff
@@ -26,6 +26,7 @@ Enhancements
 Bug Fixes
 +++++++++
 * Fix old MAC addresses being left on port-security enabled ports in a RADIUS + port-security environment
+* Fix firewall rule that allows httpd.portal to be reach on management IP when pre-registration enabled
```
Comment (extrafu) [inline NEWS.asciidoc]: to be reached from the mangement IP

**Comment by julsemaan** [comment]
Agreed,

Other than that, it looks good to me

**Comment by dwlfrth** [comment]
Then you know what to do ;)

**Pull request merged**